### ISSUE TITLE
Scene does not change state

### ISSUE THREAD

**Issue by f00dagi** [icanos/hassio-plejd#85]
Hi!

I have created a scene named 'dubbel' in the Plejd app which simply adjust a light source to it's lowest brightness through a double click on a physical light switch (DIM-01). That works fine.

After home assistant reboot I get a switch.dubbel entity which has the state 'off'. The issue is that I cannot get the state to change from 'off' to 'on' either through double click on the physical light switch or through changing the state within home assistant. It remains at 'off' and the scene is not triggered.

In other words creating a scene in the Plejd app and make it show in home assistant is easy but I cannot by any means trigger the scene.

Thanks for your efforts.

hassio-plejd: 0.4.5
home assistant: 0.110.0

**Comment by icanos** [comment]
Hi,

I've looked into this and it seems that Plejd doesn't send a notification back when a scene is triggered, since that is just a tap like action. Therefore, a switch isn't the correct entity in HA. Not sure what it should be.

I will however look into why its not triggered correctly.

**Comment by icanos** [comment]
Could you update to latest and try again?

**Comment by f00dagi** [comment]
I have updated to 0.4.7 and 0.111.1 and tried everything I can think of but I cannot get the switch to change state from 'off' to 'on'.

![image](https://user-images.githubusercontent.com/56124786/84544684-0ee85800-acfe-11ea-9a7b-c113e05b5505.png)
![image](https://user-images.githubusercontent.com/56124786/84544653-fd06b500-acfd-11ea-85f0-d5413a51e17b.png)

**Comment by icanos** [comment]
I don't think I'll be able to fix that since scenes in Plejd doesn't trigger any BLE event (that I know of at least).

**Comment by JohnPhoto** [comment]
See my comment in #134, short version: change setting in scene to ”1%” instead of ”min” and restart this plugin and HA. The conf-sync seems to be unable to handle the min setting in HA which makes HA send empty message on brigthness, which in turn crashes the write to lamp.

**Comment by SweVictor** [comment]
Scene handling have been completely rewritten in #179, merged to develop branch. Scenes are now registered as scenarios in Home Assistant and there is a device automation channel that informs HA of any scenes triggered. 

Please test the latest develop version (or wait for next release) and open a new issue if problems persists. As @icanos mentions, if no BLE command is emitted you won't be able to pick it up in HA. You can see BLE commands using verbose or silly logging in the addon configuration
